src/main/java/com/alkemy/ong/model/Category.java

```diff
@@ -0,0 +1,33 @@
+package com.alkemy.ong.model;
+
+import com.sun.istack.NotNull;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+
+@Entity
+@Table(name = "categories")
+@Getter
+@Setter
+@SQLDelete(sql = "UPDATE categories SET deleted = true WHERE id=?")
+@Where(clause = "softDelete=false")
+public class Category {
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    private Long id;
+    @NotNull
```

Comment:
Para el caso que la variable sea not null es conveniente agregar un mensaje de error, esto tambien lo podemos hacer mediante annotations